api/specs/storage/scripts/storage.py

```diff
@@ -340,17 +340,16 @@ async def delete_folders_of_project(
 
 
 @app.post(
-    f"/{API_VTAG}/simcore-s3/files/metadata:search",
+    f"/{API_VTAG}/simcore-s3/files/metadata:search_owned",
```

Comment:
The path has to be a hierarchy of **nouns** and the operation has to be a **verb**. 
Now, the resource is `simcore-s3/files/metadata`. Typically, if you want to filter this resource e.g. by a give group, you use the query. E.g. `GET simcore-s3/files/metadata?group='executables'`
Now, if you want to **fix** the filter  I would rather freeze it in the path name
`GET simcore-s3/executable-files/metadata`

Here IMO it would I would do either
- `POST /simcore-s3/files/metadata:search?kind=owned` and reject any other `kind`
- `POST /simcore-s3/owned-files/metadata:search`

Reply:
I would vote for the kind=owned, the second option seems less readable.

Reply:
OK, I go for the first version